tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/EnterpriseLibraryMsSqlTests.cs

```diff
@@ -1,29 +1,39 @@
 // Copyright 2020 New Relic, Inc. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using NewRelic.Agent.IntegrationTestHelpers;
-using NewRelic.Agent.Tests.TestSerializationHelpers.Models;
+using NewRelic.Agent.IntegrationTestHelpers.RemoteServiceFixtures;
 using NewRelic.Agent.IntegrationTests.Shared;
+using NewRelic.Agent.Tests.TestSerializationHelpers.Models;
 using NewRelic.Testing.Assertions;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace NewRelic.Agent.UnboundedIntegrationTests.MsSql
 {
     [NetFrameworkTest]
-    public class EnterpriseLibraryMsSqlTests : NewRelicIntegrationTest<RemoteServiceFixtures.MsSqlBasicMvcFixture>
+    public class EnterpriseLibraryMsSqlTests : NewRelicIntegrationTest<ConsoleDynamicMethodFixtureFW462>
     {
-        private readonly RemoteServiceFixtures.MsSqlBasicMvcFixture _fixture;
+        private readonly ConsoleDynamicMethodFixture _fixture;
+        private readonly string _expectedTransactionName;
+        private readonly string _tableName;
 
-        public EnterpriseLibraryMsSqlTests(RemoteServiceFixtures.MsSqlBasicMvcFixture fixture, ITestOutputHelper output) : base(fixture)
+        public EnterpriseLibraryMsSqlTests(ConsoleDynamicMethodFixtureFW462 fixture, ITestOutputHelper output) : base(fixture)
         {
-            MsSqlWarmupHelper.WarmupMsSql();
-
             _fixture = fixture;
             _fixture.TestLogger = output;
+            _expectedTransactionName = $"OtherTransaction/Custom/MultiFunctionApplicationHelpers.NetStandardLibraries.MsSql.EnterpriseLibraryClientExerciser/MsSql";
+
+            _tableName = Utilities.GenerateTableName();
+
+            _fixture.AddCommand($"EnterpriseLibraryClientExerciser CreateTable {_tableName}");
+            _fixture.AddCommand($"EnterpriseLibraryClientExerciser MsSql {_tableName}");
+            _fixture.AddCommand($"EnterpriseLibraryClientExerciser Wait 5000"); // TBD if this is really necessary
```

Comment:
What was the result of the TBD? ;-)

Reply:
Too be honest, never checked without it.  It was a copy-paste from the other mssql test.